app/models/usergroup.rb

```diff
@@ -73,6 +73,14 @@ def remove_users(userlist)
     self.users = self.users - User.where(:lower_login => userlist.map(&:downcase))
   end
 
+  def allowed_to?(action)
```

Comment:
If this is just meant to exist because of https://github.com/theforeman/foreman/pull/3556/files#diff-8c3f2a547e7f7d66e1756971a1ef20c2R264 - let's just not have it, it's kind of unnecessarily changing existing behavior